plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/glue/v1/DeltaLakeGlueV1MetastoreTableOperations.java

```diff
@@ -63,7 +63,7 @@ public void commitToExistingTable(SchemaTableName schemaTableName, long version,
         ImmutableMap.Builder<String, String> parameters = ImmutableMap.builder();
         parameters.putAll(getTableParameters(currentTable));
         parameters.putAll(tableMetadataParameters(version, schemaString, tableComment));
-        tableInput.withParameters(parameters.buildOrThrow());
+        tableInput.withParameters(parameters.buildKeepingLast());
```

Comment:
May be we can handle it by filtering out old TRINO_LAST_TRANSACTION_VERSION from a previous step instead ?
Using `buildKeepingLast` relies on keys being inserted in certain order which somebody might change without knowing we relied on it to keep only latest TRINO_LAST_TRANSACTION_VERSION

Reply:
Let me handle in a follow-up because `buildKeepingLast()` is consistent with other metatore logic at this time.